Packs/CheckPointHEC/Integrations/CheckPointHEC/CheckPointHEC.py

```diff
@@ -362,14 +392,256 @@ def send_notification(self, entity: str, emails: List[str]):
+        return self._call_api('GET', 'sectools/anomaly/exceptions')
+
+    def create_anomaly_exceptions(self, request_json: dict, added_by: str = None):
+        request_data = self._strip_none({
+            'requestJson': request_json,
+            'addedBy': added_by
+        })
+        payload = {
+            'requestData': request_data
+        }
+        return self._call_api('POST', 'sectools/anomaly/exceptions', json_data=payload)
+
+    def delete_anomaly_exceptions(self, rule_ids: List[str]):
+        request_data = {
+            'ruleId': rule_ids
+        }
+        payload = {
+            'requestData': request_data
+        }
+        return self._call_api('DELETE', 'sectools/anomaly/exceptions', json_data=payload)
+
+    def get_ctp_lists(self):
+        return self._call_api('GET', 'sectools/click_time_protection/exceptions')
+
+    def get_ctp_list(self, list_id: str):
+        return self._call_api('GET', f'sectools/click_time_protection/exceptions/{list_id}')
+
+    def get_ctp_list_items(self):
+        return self._call_api('GET', 'sectools/click_time_protection/exceptions/items')
+
+    def get_ctp_list_item(self, item_id: str):
+        return self._call_api('GET', f'sectools/click_time_protection/exceptions/items/{item_id}')
+
+    def create_ctp_list_item(self, list_id: str, list_item_name: str, created_by: str):
+        request_data = {
+            'listId': list_id,
+            'listItemName': list_item_name,
+            'createdBy': created_by}
+        payload = {
+            'requestData': request_data
+        }
+        return self._call_api('POST', 'sectools/click_time_protection/exceptions/items', json_data=payload)
+
+    def update_ctp_list_item(self, item_id: str, list_id: str, list_item_name: str, created_by: str):
+        request_data = {
+            'listId': list_id,
+            'listItemName': list_item_name,
+            'createdBy': created_by}
+        payload = {
+            'requestData': request_data
+        }
+        return self._call_api('PUT', f'sectools/click_time_protection/exceptions/items/{item_id}', json_data=payload)
+
+    def delete_ctp_list_item(self, item_id: str):
+        return self._call_api('DELETE', f'sectools/click_time_protection/exceptions/items/{item_id}')
+
+    def delete_ctp_list_items(self, list_item_ids: List[str]):
+        request_data = {
+            'listItemIds': list_item_ids
+        }
+        payload = {
+            'requestData': request_data
+        }
+        return self._call_api('DELETE', 'sectools/click_time_protection/exceptions/items', json_data=payload)
+
+    def delete_ctp_lists(self):
+        return self._call_api('DELETE', 'sectools/click_time_protection/exceptions')
+
 
 def test_module(client: Client):
     result = client.test_api()
-    return 'ok' if result.get('ok') else 'error'
+    scopes = result.get('responseData')
+
+    if not isinstance(scopes, list):
+        return 'error'
+
+    if len(scopes) != 1:
```

Comment:
can you please explain this if statement?

Reply:
As explained in the previous comment, the scope has to be limited to only one portal, otherwise the API calls would require the scope as a parameter and that hasn't been implemented.